docker-compose.yml

```diff
@@ -1,31 +1,34 @@
-version: '2'
+version: '3'
 
 services:
    web:
       build: .
-      entrypoint: '/opt/pulldasher/entrypoint.sh'
+      image: ${DOCKER_IMAGE_NAME}
+      depends_on:
+        - 'db'
+      env_file:
+        - "./.env.db"
```

Comment:
Seems like a bit of trouble to separate out the db env. The default behavior is to load `.env`, so why not use that?

Reply:
Well, the thing I am calling `.env` isn't needed for the regular "production" docker run. Maybe I should call it `.env.compose`, and call this one `.env`.